tests/src/test/java/alluxio/client/fs/io/SeekableAlluxioFileOutStreamIntegrationTest.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.fs.io;
+
+import static alluxio.client.WriteType.THROUGH;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import org.junit.Test;
+
+import alluxio.AlluxioURI;
+import alluxio.client.WriteType;
+import alluxio.client.file.FileOutStream;
+import alluxio.client.file.SeekableAlluxioFileOutStream;
+import alluxio.grpc.CreateFilePOptions;
+import alluxio.util.io.PathUtils;
+
+/**
+ * Integration tests for {@code SeekableAlluxioFileOutStream}.
+ */
+public class SeekableAlluxioFileOutStreamIntegrationTest
```

Comment:
Only write(int b) is tested, can you add tests for write(byte b[], int off, int len)?

Reply:
added